test/ConfigProviderTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-fastroute for the canonical source repository
+ * @copyright Copyright (c) 2017-2018 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-fastroute/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace ZendTest\Expressive\Router;
+
+use PHPUnit\Framework\TestCase;
+use Zend\Expressive\Router\ConfigProvider;
+use Zend\Expressive\Router\RouterInterface;
+
+class ConfigProviderTest extends TestCase
+{
+    /**
+     * @var ConfigProvider
+     */
+    private $provider;
+
+    protected function setUp() : void
+    {
+        $this->provider = new ConfigProvider();
+    }
+
+    public function testInvocationReturnsArray() : array
+    {
+        $config = ($this->provider)();
+        self::assertInternalType('array', $config);
```

Comment:
$this->.... ?

Reply:
There's a forum topic discussing this currently. I think for now, we should use `$this->`, as it's consistent with every other component. If we decide to go with `self::` later as a result of that forum topic, we can apply that via a phpcs fixer.